var/spack/repos/builtin/packages/git/package.py

```diff
@@ -69,6 +76,7 @@ class Git(Package):
 
     def install(self, spec, prefix):
         env['LDFLAGS'] = "-L%s" % spec['gettext'].prefix.lib + " -lintl"
+        env['NO_GETTEXT'] = "YesPlease"
```

Comment:
still, what this line does why do you need this ? `gettext` is a dependency, flags are set above.